lib/assessment.md

```diff
@@ -41,7 +56,10 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+.big {}
+#contentPane {}
+.important.red {}
+p {} //Unless the div has a class name then its ".div p {}"
```

Comment:
Not quite. This will select all `p` elements - we only want the ones that are child elements of a `div`.